### ISSUE TITLE
Red warning: SelectionMask is accessing findDOMNode in its render()

### ISSUE THREAD

**Issue by NicoleRauch** [adazzle/react-data-grid#1440]
### Which version of React JS are you using?

✅ Officially supported ✅
- [ ] v15.4.x

⚠️ Not officially supported, expect warnings ⚠️
- [ ] v15.5.x
- [ ] v15.6.x

☣️ Not officially supported, expect warnings and errors ☣️
- [X] v16.x.x

---

### Which browser are you using?

✅ Officially supported ✅
- [ ] IE 9 / IE 10 / IE 11
- [ ] Edge
- [X] Chrome

⚠️ Not officially supported, but "should work" ⚠️
- [ ] Firefox
- [ ] Safari

---

### I'm submitting a ...

- [X] 🐛 Bug Report
- [ ] 💡 Feature Request

> 👋 Need general support? Not sure about how to use React itself, or how to get started with the Grid?
> Please do not submit support request here. Instead see

> https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md

---

### Issue Details

Please include:
- What the current behavior is
I'm getting a red warning:

![image](https://user-images.githubusercontent.com/436499/50208536-b2f37c00-0371-11e9-98af-be8895ad0704.png)

- What the desired behvaior is

No warning :-)

- (If Bug) Steps to reproduce the issue

Upgrade to React 16.

In other words, are there any plans to lift react-data-grid to React 16?

**Comment by NicoleRauch** [comment]
Are there any plans to lift react-data-grid to React 16? (or to fix the above-mentioned error in isolation?)

**Comment by NicoleRauch** [comment]
Are there any plans to lift react-data-grid to React 16? (or to fix the above-mentioned error in isolation?)

**Comment by amanmahajan7** [comment]
This error is not related to supporting `React 16`. RDG already supports React 16 as of v4.0
https://github.com/adazzle/react-data-grid/blob/master/docs/CHANGELOG.md

We just released a new alpha version (6.0.2-0) of RDG. Can you check if you are getting the same message with the latest version. Thanks

**Comment by NicoleRauch** [comment]
@amanmahajan7 Thank you for your response! The new version indeed fixes the issue. Great :-)

> RDG already supports React 16 as of v4.0

If this is the case, maybe it would be a smart move to update the issues template? ;-)

![image](https://user-images.githubusercontent.com/436499/53477052-2ac9db00-3a74-11e9-917d-144ba65cc97d.png)
